src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -392,10 +392,7 @@
       "allOf": [
```

Comment:
Can you also please update rules TA-000007 and TA-000019? They check the same thing, but for Web Apps and API Apps.

Reply:
Updated these rules and added some tests